source/_integrations/agent_dvr.markdown

```diff
@@ -0,0 +1,68 @@
+---
+title: Agent DVR
+description: Access and control Agent DVR from Home Assistant.
+ha_category:
+  - Camera
+ha_config_flow: true
+ha_release: 0.108.6
+ha_iot_class: Local Pull
+ha_codeowners:
+  - '@ispysoftware'
+ha_domain: agent_dvr
+---
+
+[Agent DVR](https://www.ispyconnect.com/download.aspx/) is a free* software DVR solution for windows 10, Mac and Linux. Agent DVR runs as a service or console application and can access and control a huge range of third party cameras with advanced motion detection including YOLO integration for object recognition. The iSpyConnect website provides secured (SSL) remote access without port forwarding needed under a subscription model.
+
+You can add Agent DVR via integrations using the IP address and port of the server e.g.,: `http://192.168.1.3:8090/`.
+
+
+## Configuration
+
+For configuration, go to the `Integrations panel` on your Home Assistant instance.
```

Comment:
What should one do after he went to the integrations panel?

Please don't code fence "Integrations panel" as it is not code.

Reply:
i've never managed to get anything working in hass using configuration.yaml and have been unable to test anything with it - would you be able to help me out here? I've only ever used the integrations menu/ UI.

Reply:
I meant; if one goes to the integration page... one would be looking at all the integrations they have installed. What is next?

Maybe explain the user to click the + sign and pick "X" from the list and follow the instructions on screen?